### ISSUE TITLE
Configuration param ignored in dotnet publish tool

### ISSUE THREAD

**Issue by yhnavein** [dotnet/sdk#3858]
I stumbled upon a strange issue in the latest .net core 3.0. It looks like `--configuration` parameter is being ignored by the publish command.

Here is the issue:
```
$ dotnet publish -c prod
Microsoft (R) Build Engine 16.3.0+0f4c62fea dla platformy .NET Core
Copyright (C) Microsoft Corporation. bla bla bla

  Ukończono przywracanie w elemencie 23,91 ms (PATH\PROJECT.csproj).
  IntegrationLayer -> PATH\PROJECT\bin\Debug\netcoreapp3.0\PROJECT.dll
  IntegrationLayer -> PATH\PROJECT\bin\Debug\netcoreapp3.0\publish\
```

I've specified `prod` configuration and as you can see in the output publish was made to the `Debug` directory. This is wrong.

I am using the same command for my other projects in dotnet core 2.2 and it was working fine.

*Observed behaviour*
Files are published to `bin\Debug\netcoreapp3.0`

*Expected behaviour*
Files are published to `bin\prod\netcoreapp3.0`

```
$ dotnet --version
3.0.100
```

**Comment by twix20** [comment]
Please make sure that your `.sln` file contains correct `project configuration ` for given `active solution configuration`.

You can check that by using `Build -> Configuration Manager` in Visual Studio

![image](https://user-images.githubusercontent.com/9865224/72322822-95ddc980-36a7-11ea-86e4-ec855fb0f0c6.png)

**Comment by yhnavein** [comment]
@twix20 It has.
But apparently it was fixed in dotnet 3.1, because now it works as it was working before 3.0.
So I am closing this one.
